packages/sui-ssr/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/ssr",
-  "version": "5.35.0",
+  "version": "5.35.1",
```

Comment:
Shouldn't be a major? As the apps relying on react-helmet will have discrepancies between server & client.

Reply:
Yep, it is a breaking change.